server/data/personClient.ts

```diff
@@ -0,0 +1,20 @@
+import type { Person } from 'approved-premises'
+import RestClient from './restClient'
+import config, { ApiConfig } from '../config'
+
+export default class PersonClient {
+  restClient: RestClient
+
+  constructor(token: string) {
+    this.restClient = new RestClient('personClient', config.apis.approvedPremises as ApiConfig, token)
+  }
+
+  async find(crn: string): Promise<Person> {
+    const response = await this.restClient.post({
+      path: `/person/search`,
```

Comment:
Yeah, I do think it should be `/people/search` actually in the spirit of RESTfulness. Will feed that back in the other PR.